dts/bindings/clock/nxp,mcxc-osc.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCXC oscillator
+
+compatible: "nxp,mcxc-osc"
+
+include: fixed-clock.yaml
+
+properties:
+  load-capacitance:
```

Comment:
please, add units here

Reply:
The property is not load capacitance directly, but rather configuration of the load. Changed the misleading property name to load-configuration (without a unit). Meaning is explained in the description of the property. @gmarull please review.

Reply:
Then this is against bindings good practices, use meaningful units.

Reply:
Changed to femtofarads.

Reply:
@gmarull could you review the changes, please?